<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OAControl</class>
 <widget class="QMainWindow" name="OAControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>761</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>851</width>
    <height>761</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>851</width>
    <height>761</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Online Analysis Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="OAControl.qrc">
    <normaloff>:/icons/mainIcon</normaloff>:/icons/mainIcon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tango_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>791</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TANGO attributes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tango_device">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>16</y>
      <width>511</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ldm/view/dynattr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>16</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="tango_status">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>19</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
background-color: red;
color: white</string>
    </property>
    <property name="text">
     <string>UNKNOWN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>791</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="oa_server">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>331</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>ldm/postprocessing/ws</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>23</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>OA Server</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="oa_status">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>23</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
background-color: red;
color: white</string>
     </property>
     <property name="text">
      <string>UNKNOWN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="reload_oa_button">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Reload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_oa_button">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="oa_queue">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>286</x>
       <y>54</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Queue length</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="oa_error">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>53</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
background-color: red;
color: white</string>
     </property>
     <property name="text">
      <string>UNKNOWN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>24</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Proc. status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>53</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Error status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>OA Status</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="plot_selection">
       <property name="text">
        <string>Plot selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_selection">
       <property name="text">
        <string>Save selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveall_button">
       <property name="text">
        <string>Save all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>Reset all</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>477</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Error log</string>
    </property>
   </widget>
   <widget class="QListView" name="error_log">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>791</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_reload">
   <property name="text">
    <string>Reload configuration</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tango_device</tabstop>
  <tabstop>connect_button</tabstop>
  <tabstop>oa_server</tabstop>
  <tabstop>connect_oa_button</tabstop>
  <tabstop>oa_queue</tabstop>
  <tabstop>reload_oa_button</tabstop>
  <tabstop>tango_list</tabstop>
  <tabstop>plot_selection</tabstop>
  <tabstop>save_selection</tabstop>
  <tabstop>saveall_button</tabstop>
  <tabstop>clear_button</tabstop>
 </tabstops>
 <resources>
  <include location="OAControl.qrc"/>
 </resources>
 <connections/>
</ui>
